src/components/TodoList.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+export default (props) => {
```

Comment:
You should name the components you create instead of exporting out anonymous functions. If something breaks down in this, you'll have a hell of a time running around stack trace errors.